src/components/ActionControl/__snapshots__/ActionControl.test.tsx.snap

```diff
@@ -1,32 +1,46 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
-exports[`ActionControl renders 1`] = `
+exports[`ActionControl > renders 1`] = `
 <DocumentFragment>
   <form
-    class="root"
+    class="_root_dc87d2"
```

Comment:
Do you know if the suffix is deterministic or will it change or every build?
We need to find a solution if it will change for every build

Reply:
it is generated from the filename so it should be stable: https://vitest.dev/config/#css-modules-classnamestrategy

Reply:
Seems scoped on the filename, which is a stable identifier, https://vitest.dev/config/#css-modules-classnamestrategy